api/views/phones.py

```diff
@@ -550,6 +553,61 @@ def inbound_sms(request):
     )
 
 
+@decorators.api_view(["POST"])
+@decorators.permission_classes([permissions.AllowAny])
+def inbound_sms_iq(request: Request) -> response.Response:
+    incr_if_enabled("phones_inbound_sms_iq")
+    _validate_iq_request(request)
+
+    inbound_body = request.data.get("text", None)
+    inbound_from = request.data.get("from", None)
+    inbound_to = request.data.get("to", None)
+    if inbound_body is None or inbound_from is None or inbound_to is None:
+        raise exceptions.ValidationError("Request missing from, to, or text.")
+
+    from_num = phonenumbers.format_number(
+        phonenumbers.parse(inbound_from, "US"),
+        phonenumbers.PhoneNumberFormat.E164,
+    )
+    single_num = inbound_to[0]
+    relay_num = phonenumbers.format_number(
+        phonenumbers.parse(single_num, "US"), phonenumbers.PhoneNumberFormat.E164
+    )
+
+    relay_number, real_phone = _get_phone_objects(relay_num)
+    _check_remaining(relay_number, "texts")
+
+    # TODO: handle replies
+
+    number_disabled = _check_disabled(relay_number, "texts")
+    if number_disabled:
+        return response.Response(
+            status=200,
+            template_name="twiml_empty_response.xml",
```

Comment:
Does IQ implement twiml? Seems unlikely.

Reply:
They don't. Hmm I'll just change it to return an empty response 200 until we learn that doesn't work for some reason.